src/CoinbaseSmartWallet.sol

```diff
@@ -209,6 +210,10 @@ contract CoinbaseSmartWallet is MultiOwnable, UUPSUpgradeable, Receiver, ERC1271
         }
     }
 
+    function upgradeToAndCall(address newImplementation, bytes memory data) public payable override {
+        ERC1967Utils.upgradeToAndCall(newImplementation, data);
```

Comment:
this is not actually safe, but OZ is a little annoying to work with here. Doesn't have a clone1967

Reply:
For context, this override  is unsafe but temporarily necessary to test gas.

OZ's `Clones` library [deploys 1167 minimal proxies](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/proxy/Clones.sol#L9) but our smart account implementation is a `UUPSUpgradeable` contract which, when calling `upgradeToAndCall`, uses the 1967 standard [to check](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/proxy/utils/UUPSUpgradeable.sol#L99) if the `IMPLEMENTATION_SLOT` is set to `__self`. This reverts as 1167 does not use `IMPLEMENTATION_SLOT`.